rollup.js

```diff
@@ -1,20 +1,27 @@
 Chomp.addExtension('chomp@0.1:npm');
 
 Chomp.registerTemplate('rollup', function (task) {
-  const { outdir = 'dist', entries, sourceMaps = true, autoInstall } = task.templateOptions;
+  const { outdir = 'dist', entries, sourceMaps = true, autoInstall, plugins = [] } = task.templateOptions;
   if (!Array.isArray(entries) || !entries.every(entry => typeof entry === 'string'))
     throw new Error("'entries' must be an array of string module input paths for the Rollup plugin.");
   const targets = entries.map(entry => outdir + '/' + entry.split('/').pop());
   return [{
     name: task.name,
     deps: [...task.deps, ...ENV.CHOMP_EJECT ? ['npm:install'] : ['node_modules/rollup']],
```

Comment:
We should also add the deps here:

```suggestion
    deps: [...task.deps, ...ENV.CHOMP_EJECT ? ['npm:install'] : ['node_modules/rollup', ...plugins.map(p => `node_modules/${p}`]],
```